docs/content/sre/machinepools/machinepools.md

```diff
@@ -22,6 +22,7 @@ The overall minimum resource requirements are:
 | Worker | 4 x 16 x 100 | 3 | 12 | 48 | 300 | Yes |
 | **Total minimum** | | 9 | 36 | 208 | 900 | |
 | **Total recommended** | | 11 | 44 | 240 | 1100 | |
+
```

Comment:
without this break the heading is not displayed but meshed together into the table